### PR TITLE
minor edits

### PR DESCRIPTION
**Pull request by rosemarymarano**
Very minor edits, however, two items of note:
1. Monica has asked whether this doc, which tracks changes to the Operator, should list the changes for v1.
2. This doc has cross-references to the Name Changes doc. Tom M has proposed getting rid of that doc for GA, and would impact the content of this doc.

### CODE REVIEW

File: site/recent-changes.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+# Recent changes to the Oracle WebLogic Server Kubernetes Operator
 
-This page tracks recent changes to the operator, especially ones that introduce backward incompatibilities.
+This document tracks recent changes to the operator, especially ones that introduce backward incompatibilities.
 
-| Date | Introduces Backward Incompatibilities | Change |
+| Date | Introduces backward incompatibilities | Change |
 | --- | --- | --- |
 | March 20, 2018 | yes | See [Name Changes](name-changes.md).  Several files and input parameters have been renamed.  This affects how operators and domains are created.  It also changes generated Kubernetes artifacts, therefore customers must recreate their operators and domains.
 | April 4, 2018 | yes | See [Name Changes](name-changes.md).  Many Kubernetes artifact names and labels have changed. Also, the names of generated YAML files for creating a domain's PV and PVC have changed.  Because of these changes, customers must recreate their operators and domains.
```
Comment (mriccell) [inline site/recent-changes.md]: | --- | --- | --- |
 | May 7, 2018 | no | Add support for Dynamic Clusters, add support for Apache HTTP Server, add support for Voyager Ingress Controller, add support for PV in NFS storage for multi node environments.

File: site/recent-changes.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Recent Changes to the Oracle WebLogic Server Kubernetes Operator
+# Recent changes to the Oracle WebLogic Server Kubernetes Operator
 
-This page tracks recent changes to the operator, especially ones that introduce backward incompatibilities.
+This document tracks recent changes to the operator, especially ones that introduce backward incompatibilities.
 
-| Date | Introduces Backward Incompatibilities | Change |
+| Date | Introduces backward incompatibilities | Change |
 | --- | --- | --- |
 | March 20, 2018 | yes | See [Name Changes](name-changes.md).  Several files and input parameters have been renamed.  This affects how operators and domains are created.  It also changes generated Kubernetes artifacts, therefore customers must recreate their operators and domains.
```
Comment (mriccell) [inline site/recent-changes.md]: If we are removing the name-changes.md this reference should be removed from the comment.

Comment (rosemarymarano) [reply site/recent-changes.md]: True, but we didn't remove name-changes.md. Tom suggested it. He asked for Loren's opinion (so I added Loren as a reviewer), and, Ryan already approved and merged my edits in that file.

File: site/recent-changes.md (hunk 1 above)
Comment (rosemarymarano) [reply site/recent-changes.md]: Done.

Comment (moreaut) [reply site/recent-changes.md]: Rosemary, as part of your change, could you remove the references to name-changes.md from recent-changes.md, and git rm name-changes.md too?  Thanks, -Tom

Comment (rosemarymarano) [reply site/recent-changes.md]: Will do.

Comment (rosemarymarano) [reply site/recent-changes.md]: Tom, do you want me to remove the cross references to name-changes.md only or to also remove the accompanying text, for example, "Several files and input parameters have been renamed.  This affects how operators and domains are created.  It also changes generated Kubernetes artifacts, therefore customers must recreate their operators and domains." Similar text occurs in the March 20 and April 4 changes.

**Comment by mriccell** [comment]
Rosemary there is a conflict here, probably a change to this file was merged and you need to follow the steps described in "Use the command line"

**Pull request merged**